### DISCUSSION TITLE
Need Some Clearity on following Questions.

### DISCUSSION THREAD

**Discussion by Sagar9921** [seaweedfs/seaweedfs#6315] [Q&A]
1] for every service like master service,filer service and volume service we need to set up tls and certificates  saperately? and does seaweedfs have the ui for volume service?
2] If my organization is using seaweedfs for more number of employee then what about the security of data?
3]if my 1 volume service is full then what could be the scenerio to increase it apart from increasing the service directly? is it possible to set memory of volume service directly through the configuration that we do in yaml file?
@chrislusf @chrisluuber

**Comment by chrislusf** [comment]
1. Not necessarily. Yes
2. You can use S3 APIs
3. Not clear about the question. But you can add another volume server to increase capaicty.
